src/main/java/com/cascadebot/cascadebot/commands/moderation/UnbanCommand.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package com.cascadebot.cascadebot.commands.moderation;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.ICommandMain;
+import com.cascadebot.cascadebot.commandmeta.Module;
+import com.cascadebot.cascadebot.permissions.CascadePermission;
+import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.exceptions.InsufficientPermissionException;
+
+public class UnbanCommand implements ICommandMain {
+
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length == 0) {
+            context.replyDanger("Not enough arguments (No specified member)");
+            return;
+        }
+        String targetUser = context.getMessage(0);
```

Comment:
You need to do a format check here. Use `Pattern.compile("[0-9]{17,}")` for now to match it